--- gnet-orig/configure	2009-03-10 12:21:03.000000000 +0100
+++ gnet-2.0.8/configure	2009-03-10 12:21:23.000000000 +0100
@@ -24511,7 +24511,8 @@
 See \`config.log' for more details." >&5
 echo "$as_me: error: cannot run test program while cross compiling
 See \`config.log' for more details." >&2;}
-   { (exit 1); exit 1; }; }
+#   { (exit 1); exit 1; };
+}
 else
   cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
